README.md

```diff
@@ -1,27 +1,20 @@
-# Kolla-G5K
+# Ham
 
 ## Synopsis
+
 This script deploys an OpenStack on Grid'5000 using
-[Kolla](https://wiki.openstack.org/wiki/Kolla) and allows easy customization of the system (change in the default topology, configuration parameters) and reproductible experiments.
+[Kolla](https://wiki.openstack.org/wiki/Kolla) and allows easy
+customization of the system (change in the default topology,
+configuration parameters) and reproductible experiments.
 
 ## Installation
-To install Kolla-G5K, first connect to the Grid'5000 frontend of your choice :
-and clone the repository
-
-```
-git clone https://github.com/BeyondTheClouds/kolla-g5k
-```
 
-Then install the dependencies:
+To install Ham, first connect to the Grid'5000 frontend of your
+choice and run.
 
 ```
-cd kolla-g5k
-pip install -r requirements.txt --user
+pip install git+git://github.com/BeyondTheClouds/ham@master#egg=ham
```

Comment:
I'm ok ok with the current state as long as it is documented. In the following I try to organize the usage :

- As a developper/potential contributor I need to know how to get started from the source
- From a user of the tool perspective, using the tool from the master may hurt. That's why I was suggesting to use Pypi to host tagged/verified versions. `pip install ham` sounds easier than `pip install git+git://github.com/BeyondTheClouds/ham@master#egg=ham`
- From an advanced user I can choose each one of the previous solution or install directly from the latest git revision using pip